BrainUp/BrainUp/Network/AlamofireNetworkService.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  AlamofireNetworkService.swift
+//  BrainUp
+//
+//  Created by Evgenii Zhigunov on 2/18/22.
+//
+
+import Foundation
+import Alamofire
+
+class AlamofireNetworkService: NetworkService {
+    func fetch<T>(_ request: Request, completion: @escaping (Result<T, Error>) -> Void) where T: Decodable, T: Encodable {
```

Comment:
Should we use here Encodable? I suggest use only Decodable.

Reply:
fixed